desktop/core/src/desktop/js/components/Tabs.vue

```diff
@@ -78,7 +78,7 @@
     ul {
       border-bottom: 1px solid $hue-border-color;
       list-style: none;
-      margin: 20px 12px;
+      margin: 20px 0px;
```

Comment:
If we 0 the horizontal margin we should also 0 the vertical one. We either rely on the parent padding or this ones margin.